py/torch_tensorrt/dynamo/lowering/_decompositions.py

```diff
@@ -70,12 +244,33 @@ def addmm_replacement(
     )
 
 
-@register_decomposition(torch.ops.aten.reciprocal.default, registry=DECOMPOSITIONS)
+@register_decomposition(
+    torch.ops.aten.reciprocal.default, registry=TORCH_TRT_DECOMPOSITIONS
+)
 def reciprocal_replacement(
     input_: torch.Tensor,
 ) -> torch.Tensor:
     return torch.div(1, input_)
 
 
-def get_decompositions():
-    return DECOMPOSITIONS
+def get_decompositions(
```

Comment:
The dictionary returned by `get_decompositions` is either `ENABLED_TORCH_DECOMPOSITIONS` or `CORE_ATEN_DECOMPOSITIONS_FILTERED` concatenated with our `TORCH_TRT_DECOMPOSITIONS`